tools/helper.psm1

```diff
@@ -17,8 +17,8 @@ $DotnetSDKVersionRequirements = @{
     }
 
     '6.0' = @{
-        MinimalPatch = '412'
-        DefaultPatch = '412'
+        MinimalPatch = '415'
```

Comment:
Could you please add a comment in your commit message that describes this change? E.g., `Upgrade to .NET SDK 6.0.415`?

Reply:
Added to the latest commit message, and I'll ensure it's included in the squash commit message